packages/components/src/accordion/src/Accordion.css

```diff
@@ -41,27 +42,27 @@
     border: 0;
     transition: background-color var(--o-ui-easing-duration-3) var(--o-ui-easing-focus), border-bottom-left-radius 5s linear, border-bottom-right-radius 5s linear;
     width: 100%;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
     line-height: inherit;
     font-weight: inherit;
     position: relative;
-    padding: 0 var(--o-ui-sp-2);
+    padding: 0 var(--hop-space-inset-sm);
     font-family: inherit;
     font-size: 100%;
 }
 
 /* TRIGGER | BORDERED */
 .o-ui-accordion-bordered .o-ui-accordion-trigger {
```

Comment:
not sure if these follow the inline Disclosure, if so, you may need a border: 
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/e58fda94-5f2a-4294-99d6-70196393ce2d)
I realize there is a borderless and a bordered version, but the bordered is missing a border when opened.